CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v0.23.0 - 2024-12-13
+
+### Features
+- Message builder for constructing message with carrying more information (#291) [[f760b4a]](https://github.com/aiidateam/plumpy/commit/f760b4aaf6a46bbfc13bab88e36271aab122a641)
+  - Breaking behavior: Direct message constants (`KILL_MSG`, `PAUSE_MSG`, `PLAY_MSG`, `STATUS_MSG`) are replaced with the `MessageBuilder` class methods.
+
+### Bug fixes
+- Make `Waiting.resume()` idempotent (#285) [[20e5898]](https://github.com/aiidateam/plumpy/commit/20e5898e0c9037624988fe321e784f4fe38a2e8d)
+
+### Devops
+- Make `Process.run` async (#272) [[4611154]](https://github.com/aiidateam/plumpy/commit/4611154c76ac0991bcf7371b21488f4390648c28)
+  - Breaking behavior: Changes from `Coroutine` to `Awaitable` in function signatures.
```

Comment:
If only the typing annotation was changed, then it won't affect the behavior, since typing annotation is ignored at runtime.

Reply:
It would potentially be breaking the type annotations of packages that use this library. I agree that that is unlikely to happen, but still, this is not really "devops", which should only interest developers of this package and not really consumers of it. But it is not a big deal, leave it where you prefer.